forge-gui/res/cardsfolder/upcoming/wasteful_harvest.txt

```diff
@@ -0,0 +1,8 @@
+Name:Wasteful Harvest
+ManaCost:2 G
+Types:Instant
+A:SP$ Mill | NumCards$ 5 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill five cards, then return a creature card milled this way to your hand.
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | Hidden$ True | Mandatory$ True | ChangeType$ Permanent.IsRemembered | ChangeTypeDesc$ permanent cards milled this way | ChangeNum$ 1 | SelectPrompt$ Choose a creature card milled this way | SubAbility$ DBCleanup
```

Comment:
No Mandatory for this one - SelectPrompt may also want to use "may" somehow?

This ChangeTypeDesc seems out of place